internal/osutils/autostart/autostart_linux.go

```diff
@@ -176,6 +91,11 @@ func (a *app) IsEnabled() (bool, error) {
 	return false, nil
 }
 
+func (a *App) installPath() (string, error) {
+	// do not report ~/.profile as installed, as it would be removed on uninstall.
```

Comment:
This feels like the wrong place to be addressing that logic. Please write this in a way that's explicit.